docker-entrypoint.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -eu
+
+DATASTORE_DIR="${DATASTORE_DIR:-/datastore}"
+
+# If the first argument looks like a flag, assume we want to run changedetection
+if [ "${1:0:1}" = '-' ]; then
+    set -- python ./changedetection.py -d "$DATASTORE_DIR" "$@"
+fi
+
+# If we're running as root, by default make sure process uid/gid
+# and datadir permissions are correct. This can be skipped by setting
+# KEEP_PRIVILEGES to something non-empty.
+if [ "$(id -u)" = '0' -a -z "${KEEP_PRIVILEGES:-}" ]; then
+    PUID=${PUID:-911}
+    PGID=${PGID:-911}
+
+    groupmod -o -g "$PGID" changedetection
+    usermod -o -u "$PUID" changedetection
+
+    # Look for files in datadir not owned by the correct user and chown them
+    find "$DATASTORE_DIR" \! -user changedetection -exec chown changedetection '{}' +
+
+    # Restart this script as an unprivileged user
+    exec gosu changedetection:changedetection "$BASH_SOURCE" "$@"
```

Comment:
goso wasnt installed, kills the container

Reply:
That's because you deleted the line with your merge, it was there... https://github.com/dgtlmoon/changedetection.io/pull/722/commits/48565082d4da01872cacae1c1fc0bb4346115fc3

Reply:
@lkubb oops thanks :pray: could you add it back and update the branch with master? :pray:

Reply:
Sure :)

Reply:
Thank you! then we can see if it will build across all architectures

Reply:
Just saw you also deleted the user account generation. We need that to run rootless. :) I merged master, let's see how that works.

Reply:
Now I missed a ``\`` in the ``RUN`` part, sorry. I force-pushed the correction.